### PR TITLE
refactor: automate EventTypeMessage inclusion in every message execution (backport cosmos/cosmos-sdk#13532)

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

This patch would add message event to every msgs in a tx. The event contains following attributes:
- sender
- action
- module

You may refer to the corresponding PR of the upstream.

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/finschia-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Comment by 0Tech** [comment]
Waiting for the relevant issue being closed in the upstream.

**Comment by 0Tech** [comment]
We should apply this PR as is.

File: baseapp/baseapp.go [hunk 1]
```diff
@@ -842,3 +837,22 @@ func (app *BaseApp) runMsgs(ctx sdk.Context, msgs []sdk.Msg) (*sdk.Result, error
...
+	// we set the signer attribute as the sender
+	if len(msg.GetSigners()) > 0 && !msg.GetSigners()[0].Empty() {
+		msgEvent = msgEvent.AppendAttributes(sdk.NewAttribute(sdk.AttributeKeySender, msg.GetSigners()[0].String()))
+	}
+
+	// here we assume that routes module name is the second element of the route
+	// e.g. "cosmos.bank.v1beta1.MsgSend" => "bank"
+	moduleName := strings.Split(eventMsgName, ".")
+	if len(moduleName) > 1 {
+		msgEvent = msgEvent.AppendAttributes(sdk.NewAttribute(sdk.AttributeKeyModule, moduleName[1]))
```
Comment (zemyblue) [inline baseapp/baseapp.go:854]: I think it seems to have some problem if ibc case. 
ex) `ibc.applications.fee.v1.MsgRegisterPayee`

**Review by zemyblue — APPROVED** [review]
Please check checklist in description.

**Review by jaeseung-bae — APPROVED** [review]

**Pull request merged**